ConorMFinn/CO600-SeriousCyberSec
# Add a gameplay introduction

**Description of work.**
Interacting with Ginny now brings up a conversation to explain the story a bit. 

**To test:**
Walk up to ginny and press E
Check all of the text boxes in the coversation for errors and clarity. 

Code Review Requireed on:
- [x] TextBoxController.cs
- [x] NpcInteraction.cs

Fixes #77
